### PR TITLE
chore: update to use plugin api app.getSelfPath

### PR DESCRIPTION
**Pull request by sbender9**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -199,7 +199,7 @@ module.exports = function (app) {
   }
 
   function setKey (info, dest_key, source_key) {
-    var val = _.get(app.signalk.self, source_key)
+    var val = app.getSelfPath(source_key)
     if (val !== undefined) info[dest_key] = val
```
Comment (tkurki) [inline index.js:203]: Setting values like this won’t work in the future either. Do we need  `setSelfPath`?

A bit different matter, so does not prevent merging this.

Comment (sbender9) [reply index.js:203]: I don't think so. I've never seen anything that sets values that way. Should be done with delta's, right?

Comment (sbender9) [reply index.js:203]: Actually, might be a nice shorthanded way for plugins to send deltas.

Comment (tkurki) [reply index.js:203]: Deltas would be the way, yes.

**Pull request merged**